cost/aws/elb/clb_unused/AWS_Delete_Unused_CLB.pt

```diff
@@ -108,6 +108,7 @@ script "js_aws_clb_filter_map", type: "javascript" do
 	
   result "content"
   code <<-EOS
+  var param_exclude_tags_lower=param_exclude_tags.toString().toLowerCase().split(',');
```

Comment:
This still results in the same corner cases caused by the `,` delimiter actually being in the string. I would say loop over the param_exclude_tags array and insert a lowercase string in the temporary array.

Reply:
Now we had taken care this one with loop. Thanks.